test/vanilla/Expected/AcceptanceTests/ModelFlattening/modelflattening/aio/operations/_auto_rest_resource_flattening_test_service_operations.py

```diff
@@ -454,6 +454,7 @@ async def post_flattened_simple_product(
         product_id: str,
         description: Optional[str] = None,
         max_product_display_name: Optional[str] = None,
+        capacity: Optional[str] = "Large",
```

Comment:
as you'll notice, the default for optional constants in the method signature is a value. This has to do with [this](https://github.com/Azure/autorest.python/issues/893) issue. Still need to find a good non-breaking way around this